content/spin/v2/ai-sentiment-analysis-api-tutorial.md

```diff
@@ -294,25 +281,27 @@ The Rust source code for this sentiment analysis example uses serde. There are a
 
 ```toml
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0.85"
+serde_json = "1.0"
 ```
 
 Once you have added serde, as explained above, modify your `src/lib.rs` file to match the following content:
 
+<!-- TODO - Awaiting https://github.com/fermyon/spin/issues/1995 -->
+
 ```rust
 use std::str::FromStr;
 
 use anyhow::Result;
 use spin_sdk::{
-    http::{Params, Request, Response, Router},
+    http::{IntoResponse, Json, Params, Request, Response, Router},
     http_component,
     key_value::Store,
     llm::{infer_with_options, InferencingModel::Llama2Chat},
 };
 
 use serde::{Deserialize, Serialize};
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Serialize)]
```

Comment:
Why does this now need `Serialize`?

Reply:
It does not, might have been an intermediate step where this was needed while trying to get the new SDK working. Removing.